### PR TITLE
CVSB-5561

### PR DESCRIPTION
**Pull request by vladddragomir**

### CODE REVIEW

File: src/modules/logs/logs.model.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ export type Log = {
   message: string,
   timestamp: number,
   [propName: string]: any,
+  unauthenticated?: boolean
```
Comment (agoncearenco) [inline src/modules/logs/logs.model.ts]: Are you sure you need to declare the unauthenticated property here? I think the additional properties are allowed by [propName: string]: any [(described here)](https://www.typescriptlang.org/docs/handbook/interfaces.html#excess-property-checks)

Comment (agoncearenco) [inline src/modules/logs/logs.model.ts]: I also think making properties optional is not necessary as our project is not configured to verify unset properties. We can leave unset any property and typescript will not complain about it.

Comment (vladddragomir) [reply src/modules/logs/logs.model.ts]: yes, i think you're right. will update the code

Comment (vladddragomir) [reply src/modules/logs/logs.model.ts]: if i remove unauthenticated property, will also solve this

Comment (vladddragomir) [reply src/modules/logs/logs.model.ts]: done

File: src/providers/global/auth.service.ts [hunk 2]
```diff
@@ -159,3 +188,9 @@ export class AuthService {
     return userRoles.some(role => neededRoles.indexOf(role) >= 0);
   }
 }
+
+import { FirebaseLogsService } from '../firebase-logs/firebase-logs.service';
+import { Log, LogsModel } from "../../modules/logs/logs.model";
+
+import * as logsActions from "../../modules/logs/logs.actions";
+import { Store } from "@ngrx/store";
```
Comment (agoncearenco) [inline src/providers/global/auth.service.ts]: Could you please move these imports at the beginning of the file?

Comment (vladddragomir) [reply src/providers/global/auth.service.ts]: done! no idea why they were there

**Pull request merged**